portfolio/src/App.jsx

```diff
@@ -1,157 +1,145 @@
-              <div className="flex justify-evenly mt-12">
-                <button className="px-6 py-3 text-lg bg-gradient-to-t from-orange-500 rounded-full to-yellow-500 hover:from-orange-700 hover:to-yellow-700">
-                  Live preview
-                </button>
-                <button className="px-6 py-3 border rounded-full hover:border-orange-500 hover:text-yellow-500">
-                  Checkout github
-                </button>
-              </div>
-            </div>
-            <div className="border border-gray-500 rounded-md p-5">
-              {/*https://tfkjosh.github.io/Password-generator/*/}
-              <a href="https://tfkjosh.github.io/Password-generator/" target="_blank" class="project 3"></a>
-              <img src={Project3} alt="https://tfkjosh.github.io/Password-generator/" className="w-full h-auto" />
-              <h3 className="text-2xl font-semibold mt-8">
-              This application is designed for an employee to generate a random password based on criteria they've selected.
-              </h3>
-              <p className="text-gray-400 text-sm mt-4">https://github.com/tfkjosh/Password-generator</p>
-              <div className="flex justify-evenly mt-12">
-                <button className="px-6 py-3 text-lg bg-gradient-to-t from-orange-500 rounded-full to-yellow-500 hover:from-orange-700 hover:to-yellow-700">
-                  Live preview
-                </button>
-                <button className="px-6 py-3 border rounded-full hover:border-orange-500 hover:text-yellow-500">
-                  Checkout github
-                </button>
-              </div>
-            </div>
-            <div className="border border-gray-500 rounded-md p-5">
-              {/*https://pwa-challenge1-b508091ee337.herokuapp.com/*/}
-              <a href="https://pwa-challenge1-b508091ee337.herokuapp.com/" target="_blank" class="project 4"></a>
-              <img src={Project4} alt="https://pwa-challenge1-b508091ee337.herokuapp.com/" />
-              <h3 className="text-2xl font-semibold mt-8">
-              This application features a number of data persistence techniques, insuring the application runs regardless of browser supported function. 
-              </h3>
-              <p className="text-gray-400 text-sm mt-4">https://github.com/tfkjosh/PWA-Challenge</p>
-              <div className="flex justify-evenly mt-12">
-                <button className="px-6 py-3 text-lg bg-gradient-to-t from-orange-500 rounded-full to-yellow-500 hover:from-orange-700 hover:to-yellow-700">
-                  Live preview
-                </button>
-                <button className="px-6 py-3 border rounded-full hover:border-orange-500 hover:text-yellow-500">
-                  Checkout github
-                </button>
-                </div>
-              </div>
+            <div className="flex flex-col sm:flex-col gap-12 mt-11">
+              {projects.map(
```

Comment:
Another instance of using a `map` function to iterate over a list/array of data to make all of the repeating UI elements consistent and very easy to update